docs/developers/torch_add_a_new_converter.md

```diff
@@ -0,0 +1,89 @@
+# How To Add a New Torch Op Converter
+
+## 1. Get the Operator's Function Schema
```

Comment:
As the comment: https://github.com/alibaba/BladeDISC/pull/17#discussion_r775353050 
please remove the number on the title.